openedx_learning/core/contents/api.py

```diff
@@ -31,8 +40,11 @@ def create_raw_content(
     return raw_content
 
 
-def create_text_from_raw_content(raw_content, encoding="utf-8-sig"):
-    text = codecs.decode(raw_content.file.open().read(), "utf-8-sig")
+def create_text_from_raw_content(raw_content: RawContent, encoding="utf-8-sig") -> TextContent:
+    """
+    Create a new TextContent instance for the given RawContent.
+    """
+    text = codecs.decode(raw_content.file.open().read(), encoding)
```

Comment:
This was a bug (I assume): the API accepted an `encoding` parameter but ignored it, and always used `"utf-8-sig"`

Reply:
Yup, ty!